src/modules/buyNFT/components/BidDialog/BidDialog.tsx

```diff
@@ -209,7 +209,7 @@ export const BidDialog = ({
             {category === 'image' ? (
               <Img className={classes.imgWrap} src={filepath} ratio="1x1" />
             ) : (
-              <VideoPlayer src={filepath} />
+              <VideoPlayer src={filepath} autoPlay={true} />
```

Comment:
Also you can use a short variant
```suggestion
              <VideoPlayer src={filepath} autoPlay />
```

Reply:
ok